Prox/Prox.xcodeproj/project.pbxproj

```diff
@@ -488,7 +503,7 @@
 				TargetAttributes = {
 					7B29E7BE1DA3EC360099AF38 = {
 						CreatedOnToolsVersion = 8.0;
-						DevelopmentTeam = 9G8J6YA743;
```

Comment:
This happens when I deploy to device using the Prox Enterprise scheme. I keep forgetting to check for it when I check stuff in. It is part of Apple's automatic codesigning thing that is a very new thing, but essential for the smooth running of CI.